lib/script.c

```diff
@@ -346,6 +346,8 @@ ExprList *flux_script_parse_list(VectorCursor *token_cursor, VectorCursor *list_
 
   // At this point, assume we've reached TokenKindNone
   PARSE_LOG(list_cursor->current_item, "EXIT list parse\n");
+
+  return NULL;
```

Comment:
I think I'll get rid of the return type on this function, I don't even use the return value anywhere anymore.

Reply:
Makes sense :-)